composer.json

```diff
@@ -10,7 +10,7 @@
 	"require-dev": {
 		"squizlabs/php_codesniffer": "~2.2",
 		"phpmd/phpmd": "~2.3",
-		"ockcyp/covers-validator": "~0.4.0"
+		"phpunit/phpunit": "~5.4"
```

Comment:
To make the Travis build work again for PHP 5.5 this needs to be changed to `"~4.8 || ~5.4"`.

Invalid `@covers` annotations will be break the build with PHPUnit 4.8, though.

Does this component have to be compatible with PHP 5, especially an unsupported version of PHP 5, though?